flask_project/models.py

```diff
@@ -2,40 +2,40 @@
 from sqlalchemy.orm import relationship
 from sqlalchemy.sql import func
 
-
 db = SQLAlchemy()
 
```

Comment:
Please avoid this type of comments because you can restore something whenever you want from the git.

Reply:
@mpobozhe, do you mean I should delete the User and Role model instead of commenting that out?